src/modules/dashboard/components/ActionsPage/TransactionHash/Hash.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {
+  defineMessages,
+  FormattedMessage,
+  MessageDescriptor,
+} from 'react-intl';
+
+import styles from './TransactionHash.css';
+
+const MSG = defineMessages({
+  fallbackTitle: {
+    id: 'dashboard.ActionsPage.TransactionHash.Hash.fallbackTitle',
+    defaultMessage: 'Transaction hash',
+  },
+});
+
+const displayName = 'dashboard.ActionsPage.TransactionHash.Hash';
+
+interface Props {
+  transactionHash: string;
+  title?: MessageDescriptor;
+}
+
+const Hash = ({ transactionHash, title = MSG.fallbackTitle }: Props) => (
+  <>
+    <p className={styles.title}>
+      <FormattedMessage {...title} />
```

Comment:
did you test how it looks and works when title is undefined?

Reply:
When the `title` is undefined it falls back to `MSG.fallbackTitle`